opm/simulators/utils/MPISerializer.hpp

```diff
@@ -52,7 +52,15 @@ class MpiSerializer : public Serializer<Mpi::Packer> {
             try {
                 this->pack(data);
                 m_comm.broadcast(&m_packSize, 1, root);
-                m_comm.broadcast(m_buffer.data(), m_packSize, root);
+                const int maxChunkSize = std::numeric_limits<int>::max();
+                std::size_t remainingSize = m_packSize;
+                std::size_t pos = 0;
+                while (remainingSize > maxChunkSize) {
+                    m_comm.broadcast(m_buffer.data()+pos, maxChunkSize, root);
+                    pos += maxChunkSize;
+                    remainingSize -= maxChunkSize;
+                }
+                m_comm.broadcast(m_buffer.data()+pos, static_cast<int>(remainingSize), root);
```

Comment:
Maybe put this into a separate function for reuse bwlow?

Reply:
Good point, will do.